android/src/com/frostwire/android/gui/views/AdMenuItemView.java

```diff
@@ -0,0 +1,77 @@
+/*
+
+ * Created by Angel Leon (@gubatron), Alden Torres (aldenml),
+ * Marcelina Knitter (@marcelinkaaa)
+ * Copyright (c) 2011-2016, FrostWire(R). All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.frostwire.android.gui.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.RelativeLayout;
+import android.widget.TextView;
+import com.frostwire.android.R;
+
+import java.util.Random;
+
+
+/**
+ * @author gubatron
+ * @author aldenml
+ *
+ */
+public class AdMenuItemView extends RelativeLayout {
+
+    private TextView textThumbnail;
+    private ImageView imageThumbnail;
+    private TextView textHeadline;
+    private TextView textSubtitle;
+
+    public AdMenuItemView(Context context, AttributeSet set) {
+        super(context, set);
+    }
+
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        View.inflate(getContext(), R.layout.view_ad_menuitem, this);
+
+        textHeadline = (TextView) findViewById(R.id.view_ad_menu_item_headline);
+        textSubtitle = (TextView) findViewById(R.id.view_ad_menu_item_subtitle);
+        textThumbnail = (TextView) findViewById(R.id.view_ad_menu_item_thumbnail);
+        imageThumbnail = (ImageView) findViewById(R.id.view_ad_menu_item_thumbnail_image);
+
+        Random myRand = new Random();
+        myRand.nextInt();
+        boolean isEven = (myRand.nextInt() % 2) == 0;
+
+        if (isEven == true) {
```

Comment:
next time just do:

`if (isEven) {`

no need to compare to `true` the `if` always checks against `true`